FlowCrypt/Common/Constants.swift

```diff
@@ -13,11 +13,11 @@ enum Constants {
 
     enum EmailConstant {
         static let recoverAccountSearchSubject = [
-            "Your FlowCrypt Backup",
-            "Your CryptUp Backup",
-            "Your CryptUP Backup",
-            "CryptUP Account Backup",
-            "All you need to know about CryptUP (contains a backup)",
+            "1Your FlowCrypt Backup",
```

Comment:
This should not be checked into the repo. If you need to test situation without a backup, you should log into another account.

Reply:
this is still WIP

Reply:
I'm mentioning it because last time it was also there when PR was approved for review. It could have slipped into production code if I didn't notice it. It's safer to just use more accounts. I can share some accounts to use.

Reply:
Safer for sure. 
Would be nice to have some other accounts for testing.